### DISCUSSION TITLE
Aspect Ratio Classes Not working with tailwind safe list config

### DISCUSSION THREAD

**Discussion by shrenikTrukky** [tailwindlabs/tailwindcss#6813] [Help]
My Aspect ration classes comes dynamic from api so i added it in safe list tailwind config file but it's not generate css for them so it's not working. safelist:{
"aspect-w-4",
 "aspect-h-3"
}

**Comment by Hyperlink-rc** [comment]
Hi!
I'm encountering the same issue.

I've extended the aspectRatio in tailwind configuration

`extend: {
      aspectRatio: {
        '60/97': '60 / 97',
        '2/1': '2 / 1',
        '2/3': '2 / 3',
        '3/2': '3 / 2',
        '5/2': '5 / 2',
        '16/10': '16 / 10',
      }
    },`

But this is not generating the css for them so I'm not able to use the expected classes to manage custom aspectRatios...

I'm using NextJs 14 and Tailwind 3.4.13

Thanks
